css/mobile-screen.css

```diff
@@ -0,0 +1,353 @@
+body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  top: 0;
+  left: 0;
+  font-family: Lato, Arial, sans-serif;
+  font-weight: normal;
+  font-style: normal;
+}
+
+.header-background {
+  top: 0;
+  width: 100%;
+  height: 33em;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.header {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  height: 55px;
+  margin: 16px 0;
+}
+
+.social-media {
+  display: none;
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  width: 100%;
+  height: 55px;
+  gap: 1.8em;
+}
+
+.nav-menu {
+  display: none;
+}
+
+.nav-menu.active {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  position: absolute;
+  left: 0;
+  top: -15px;
+  width: 20.8em;
+  height: 42em;
+  background-color: rgb(0 0 0 / 90%);
+}
+
+.nav-item {
+  width: 8em;
+  height: 2em;
+  margin: 5px;
+}
+
+.nav-item-1 {
+  margin-top: 30px;
+}
+
+.nav-link {
+  color: rgb(255 255 255 / 90%);
+  text-decoration: none;
+  font-size: 1.2em;
+}
+
+.close-btn {
+  position: absolute;
+  top: 1.6em;
+  left: 20em;
+}
+
+.logo {
+  order: 1;
+  margin-left: 4.5em;
+}
+
+.burger-menu {
+  margin: 16px;
+}
+
+.header-text {
+  background: url("../img/mobile_intro_bg.png");
+  height: 30em;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.title {
+  display: flex;
+  flex-direction: column;
+  margin: 0 25px;
+  width: 304px;
+  height: 103px;
+  color: rgb(251 128 39);
+}
+
+.title span {
+  color: rgb(251 128 39 / 70%);
+}
+
+.primary-text {
+  line-height: 150%;
+  font-size: 16px;
+  height: auto;
+  text-align: justify;
+  width: 80%;
+}
+
+.meeting-date-place {
+  padding: 10px 0;
+  width: 80%;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.date {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0;
+  color: rgb(0 0 0 / 60%);
+}
+
+.place {
+  margin: 4px;
+}
+
+.main-program {
+  width: 100%;
+  height: 44em;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-start;
+  background: url("../img/pattern_bg.png");
+  background-color: #272a32;
+}
+
+.main-program-title {
+  color: #fff;
+}
+
+.separator {
+  width: 53px;
+  height: 1em;
+  border-top: 1px solid rgb(251 128 39 / 100%);
+  padding: 0 0 0.5em;
+}
+
+.program-list {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+}
```

Comment:
- [ ] Kindly make sure to align your program list cards correctly by leaving space ob both sides of each card in order to meet the provided design :wink: 

   - ### Your design
    ![image](https://user-images.githubusercontent.com/87197412/197244294-bcc017e1-e51f-486d-9c97-b403235fbfd4.png)

   - ### Provided design
   ![image](https://user-images.githubusercontent.com/87197412/197244417-b0c9f85d-fdf3-47f3-bb3a-a9aa4a42ff32.png)